src/locks/locks/blocklocks/ChestLock.ts

```diff
@@ -0,0 +1,71 @@
+import { Block, Chest as ChestState } from 'org.bukkit.block';
+import { Chest } from 'org.bukkit.block.data.type';
+import { Type } from 'org.bukkit.block.data.type.Chest';
+import { Player } from 'org.bukkit.entity';
+import { CustomItem } from '../../../common/items/CustomItem';
+import { VkItem } from '../../../common/items/VkItem';
+import { LOCK_DATA } from '../lock-items';
+import { BlockLock, BlockLockProps } from './BlockLock';
+import { DoubleChestInventory } from 'org.bukkit.inventory';
+import { Material } from 'org.bukkit';
+
+const customItems = {
+  single: new CustomItem({
+    id: 26,
+    type: VkItem.HIDDEN,
+    data: LOCK_DATA,
+  }),
+  double: new CustomItem({
+    id: 27,
+    type: VkItem.HIDDEN,
+    data: LOCK_DATA,
+  }),
+};
+
+export class ChestLock extends BlockLock {
+  private static customItems = customItems;
+  public static materials = [Material.CHEST, Material.TRAPPED_CHEST];
+
+  constructor(props: BlockLockProps) {
+    props.lockCustomItem = ChestLock.getCustomItem(props.block);
+    super(props);
+  }
+
+  public async update() {
```

Comment:
The update() in superclass is not async, which causes this method to have different return type (Promise<void> instead of void)